### PR TITLE
Local tests and formatting

### PR DESCRIPTION
**Pull request by PyromaniacalGoose**
Added more tests of moniter_home_page, along with a bit of reformtting of test files to match the general syntax.
Describe("test suite")
 it("individual test")

### CODE REVIEW

File: production/frontend/src/tests/components/injectables/calender.test.js [hunk 1]
```diff
@@ -52,7 +52,11 @@ describe("Calender render Tests", () => {
       </WebsocketContextProvider>
     </StateContextProvider>);
 
-    expect(screen.getByText("26. June")).toBeVisible();
+    let expectedDate = "26. June"; //Test taking account for local settings used by date
+    if(Intl.DateTimeFormat().resolvedOptions().locale = "da-DK") {
```
Comment (demiguard) [inline production/frontend/src/tests/components/injectables/calender.test.js]: This looks like an assignment? Is this correct?

Also Turnaries / lambda's are your friend Try:
const expectedDate = Int1.DateTimeFormat().resolvedOptions().locale === "da-DK" ? "26. juni" : "26. June";

Comment (PyromaniacalGoose) [reply production/frontend/src/tests/components/injectables/calender.test.js]: No it's not supposed to be an assignment, thanks for catching that.

File: production/frontend/src/tests/components/production_pages/monitor_pages/monitor_home_page.test.js [hunk 2]
```diff
@@ -112,4 +139,109 @@ describe("Separators test suite", () => {
     expect(res.releasedDelayed30).toEqual(0);
     expect(res.releasedDelayed30Plus).toEqual(1);
   });
+
 });
+
+describe("ActivityOrders Separators test suite", () => {
+
+  const state = new TracershopState();
+  Object.assign(state, testState);
```
Comment (demiguard) [inline production/frontend/src/tests/components/production_pages/monitor_pages/monitor_home_page.test.js:148]: These lines are technically not needed due to the TracershopState being const by convention, so you can just use testState.

But this is the niddiest of nid-pick.

Comment (PyromaniacalGoose) [reply production/frontend/src/tests/components/production_pages/monitor_pages/monitor_home_page.test.js:148]: Oh, I just matched how I saw testState being used in other test files.

Comment (demiguard) [reply production/frontend/src/tests/components/production_pages/monitor_pages/monitor_home_page.test.js:148]: as i said the niddiest of nidpicks. The most important part is that you're familiar with convention that Tracershop state is const.

I'll merge it

**Pull request merged**